tests/test_migrate.py

```diff
@@ -117,7 +114,70 @@ def test_multiple_vectors_collection(source_client, dest_client, request) -> Non
         ("local_client", "remote_client"),
         ("remote_client", "local_client"),
         ("local_client", "second_local_client"),
-        # remote - remote requires two launched Qdrant instances and is not tested
+    ],
+)
+def test_multivectors_collection(source_client, dest_client, request) -> None:
```

Comment:
one more test is required
`test_single_multivector_collection`

it should check migration of a collection with one unnamed multivector